src/main/java/edu/harvard/iq/dataverse/ingest/IngestableDataChecker.java

```diff
@@ -578,6 +579,7 @@ public String testRDAformat(MappedByteBuffer buff) {
                 String fisrt5bytes = sb.toString();
 
                 result = this.checkUncompressedFirst5bytes(fisrt5bytes);
+		}
```

Comment:
line 618 gets reported in Eclipse - the FileInputStream used to get the Channel isn't closed.